Sources/NIO/NIOCloseOnErrorHandler.swift

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2019 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+final public class NIOCloseOnErrorHandler: ChannelInboundHandler {

```

Comment:
- sorry, super nit but Jazzy renders this as is, would you mind making it `public final` (instead of `final public`)
- this needs documentation for all the `public` bits that don't come from at least `ChannelInboundHandler`, ie. the class and the `init`

Reply:
Done! Let me know if you want me to put more details in the docs.